src/main/java/org/spongepowered/common/mixin/core/world/MixinWorldServer.java

```diff
@@ -223,6 +226,8 @@
     private boolean weatherThunderEnabled = true;
     private boolean weatherIceAndSnowEnabled = true;
 
+    private WorldServer this$ = (WorldServer) (Object) this;
```

Comment:
If you double cast `(WorldServer) (Object) this`, mixins will strip the casts at runtime.